lib/periodicChecks.js

```diff
@@ -0,0 +1,49 @@
+const assignUnassignedReviewers = async (pullRequest, context) => {
+  const assignees = pullRequest.assignees.map(user => user.login);
+  const reviewers = pullRequest.requested_reviewers.map(user => user.login);
+
+  // If the PR author is assigned, then don't assign other reviewers.
+  if (assignees.includes(pullRequest.user.login)) {
+    return;
+  }
+
+  let botComment = 'Hey ';
+  const prParams = context.repo(
+    {
+      number: pullRequest.number,
+    }
+  );
+
+  const unassignedReviewers = reviewers.filter(reviewer => {
+    if (!assignees.includes(reviewer)) {
+      botComment += `@${reviewer}, `;
```

Comment:
This should not be a part of the filter function, can we generate it separately? (for readability)